openfe/setup/atom_mapping/ligandatommapper.py

```diff
@@ -14,8 +14,6 @@ class LigandAtomMapper(gufe.AtomMapper):
     Subclasses will typically implement the ``_mappings_generator`` method,
     which returns an iterable of :class:`.LigandAtomMapping` suggestions.
     """
-    _no_element_changes: bool = False #TODO: to be removed
```

Comment:
Huh was this even ever used by anything?

Reply:
It's the stub of something that was already removed